cts-java/src/test/java/org/ga4gh/cts/api/variantAnnotation/VariantAnnotationsSearchIT.java

```diff
@@ -256,6 +251,40 @@ public void checkVariantAnnotationVariants() throws AvroRemoteException {
             Variant v = client.variants.getVariant(variantId);
             assertThat(v).isNotNull();
             assertThat(v.getId()).isEqualTo(variantId);
+            assertThat(v.getStart()).isEqualTo(start);
```

Comment:
Hi to you @sarahhunt! This was actually my mistake, the variant's `.start` shouldn't be equal to the query, as was asserted, just greater than what we requested:

    assertThat(v.getStart()).isGreaterThanOrEqualTo(start);